main/src/CoffeeMachineSimulatorMain.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Scanner;
+
+public class CoffeeMachineSimulatorMain {
+
+
+     public static String getCurrentLine;
```

Comment:
@Edvard777 why this is not in main method

Reply:
done